core/cmd/remote_client_test.go

```diff
@@ -657,13 +681,14 @@ func TestClient_RemoveBridge(t *testing.T) {
 func TestClient_RemoteLogin(t *testing.T) {
 	t.Parallel()
 
-	ethClient := new(mocks.Client)
+	rpcClient, gethClient, _, assertMocksCalled := cltest.NewEthMocksWithStartupAssertions(t)
+	defer assertMocksCalled()
 	config, cleanup := cltest.NewConfig(t)
 	defer cleanup()
-	app, cleanup := cltest.NewApplicationWithConfig(t, config, ethClient)
+	app, cleanup := cltest.NewApplicationWithConfig(t, config,
+		eth.NewClientWith(rpcClient, gethClient),
```

Comment:
Could it be cleaner to return the eth client from a cltest function and pass this in directly to avoid a lot of duplicated `eth.NewClientWith` ?

Reply:
I think its nice to have the granularity of each mock with one line assembly for the NewAppXX calls